persistence-api/src/main/java/io/stargate/db/datastore/DataStore.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright DataStax, Inc. and/or The Stargate Authors
+ * Copyright The Stargate Authors
```

Comment:
For files that we took from DSE, we need to have 
```
Copyright DataStax, Inc. and/or The Stargate Authors
```

Reply:
OK I missed that, I'll add another license set.

Reply:
Can this be configured / overridden in sub-modules?.. might be worth keeping the DataStax info local to the modules that use DS code.